### PR TITLE
added ayushlohmod.json

### PR DESCRIPTION
**Pull request by ayushlohmod**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1436"></a>

### CODE REVIEW

File: public/data/ayushlohmod.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
+
```
Comment (Panquesito7) [inline public/data/ayushlohmod.json:1]: There's a missing `{` here.
```suggestion
{
```


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by ayushlohmod — COMMENTED** [review]
added "{" in file please review it!

**Review by ayushlohmod — COMMENTED** [review]
removed extra space

**Review by Panquesito7 — APPROVED** [review]
Thanks for contributing! 🚀


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Maintainable

File: public/data/ayushlohmod.json [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  "avatar": "https://github.com/ayushlohmod.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/ayushlohmod",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/ayushlohmod_",
```
Comment (kumarsonsoff3) [inline public/data/ayushlohmod.json:14]: There is a **broken link**.  Please update it to the correct one! 
image

File: public/data/ayushlohmod.json [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+      "icon": "twitter"
+    },
+    {
+      "name": "Connect with me on LinkedIn",
+      "url": "https://linkedin.com/in/ayushlohmod",
```
Comment (kumarsonsoff3) [inline public/data/ayushlohmod.json:19]: Your LinkedIn profile link doesn't bring any results. _**Please update it with the correct one!**_

image

File: public/data/ayushlohmod.json [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+    }
+  ],
+  "milestones": [
+    {
+      "title": "Started Full Stack development",
+      "date": "September 2021",
+      "icon": "NodeJs",
+      "color": "#90ee90",
+      "description": "Started full stack development courrently learning Node Js",
+      "url": "https://ayushlohmod.github.com/oscar"
```
Comment (kumarsonsoff3) [inline public/data/ayushlohmod.json:30]: There is another _**broken link**_. Please update it with the correct one! 

image

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]
All your links are broken. You should copy and paste the correct links instead of typing them manually to avoid unnecessary typos. 

Make sure you update all these links of _**Twitter, LinkedIn, Github page**_ with the correct ones as suggested to get your PR approved. 🛠
<hr/>

⚠ **_Summary:_** This code needs a Fix 🛠

**Review by ayushlohmod — COMMENTED** [review]
Fixed broken links

**Review by kumarsonsoff3 — APPROVED** [review]
LGTM 👍  Good to Goo! 🚀

**Review by emmalearnscode — APPROVED** [review]

**Pull request merged**